home/essentials/skin.dsc

```diff
@@ -1,241 +1,235 @@
 
 # $ ██ [ [c]usage:
-# - ██ | [c]/skin help                     | Shows this list in-game
-# - ██ | [c]/skin set name                 | Sets a skin by the name you saved -
-# - ██ |                                     if it doesn't exist, you'll get whatever that player's name's skin is!
-# - ██ | [c]/skin reset                    | Resets your skin to your default skin
+# - ██ | [c]/skin help                                         | Shows this list in-game
+# - ██ | [c]/skin set name                                 | Sets a skin by the name you saved -
+# - ██ |                                                                         if it doesn't exist, you'll get whatever that player's name's skin is!
+# - ██ | [c]/skin reset                                        | Resets your skin to your default skin
 # - ██ | [c]/skin rename old_name new_name | Renames a skin from old_name to the new_name specified
-# - ██ | [c]/skin save player_name         | Saves a skin by the player_name's skin
-# - ██ | [c]/skin save name url (slim)     | Saves a skin by the URL pasted - optionally making
-# - ██ |                                     it slim if specified - defaults to regular / nothing
-# - ██ | [c]/skin list                     | Lists the skins you've saved
-# - ██ | [c]/skin delete name              | Deletes a skin by the name you saved
+# - ██ | [c]/skin save player_name                 | Saves a skin by the player_name's skin
+# - ██ | [c]/skin save name url (slim)         | Saves a skin by the URL pasted - optionally making
+# - ██ |                                                                         it slim if specified - defaults to regular / nothing
+# - ██ | [c]/skin list                                         | Lists the skins you've saved
+# - ██ | [c]/skin delete name                            | Deletes a skin by the name you saved
```

Comment:
this isn't problematic but the spacing's intent here was to align the text for readability, i think this just accidentally got malformed when converting from two to four indentation
```suggestion
# - ██ | [c]/skin help                     | Shows this list in-game
# - ██ | [c]/skin set name                 | Sets a skin by the name you saved -
# - ██ |                                     if it doesn't exist, you'll get whatever that player's name's skin is!
# - ██ | [c]/skin reset                    | Resets your skin to your default skin
# - ██ | [c]/skin save player_name         | Saves a skin by the player_name's skin
# - ██ | [c]/skin save name url (slim)     | Saves a skin by the URL pasted - optionally making
# - ██ |                                     it slim if specified - defaults to regular / nothing
# - ██ | [c]/skin list                     | Lists the skins you've saved
# - ██ | [c]/skin delete name              | Deletes a skin by the name you saved
```